front/src/common/ScenarioExplorer/ScenarioExplorerModalProjectMiniCard.tsx

```diff
@@ -9,17 +10,14 @@ export default function ProjectMiniCard({
 }: MiniCardsProjectProps) {
   return (
     <div
-      className={`scenario-explorator-modal-part-itemslist-minicard project with-image ${
-        isSelected ? 'selected' : ''
-      } ${project.studies_count && project.studies_count > 0 ? '' : 'empty'}`}
+      className={cx(
+        'scenario-explorator-modal-part-itemslist-minicard project with-image',
+        isSelected && 'selected',
+        project.studies_count === 0 && 'empty'
+      )}
```

Comment:
I find classnames object syntax slightly more readable

```suggestion
      className={cx(
        'scenario-explorator-modal-part-itemslist-minicard',
        'project',
        'with-image',
        {
          selected: isSelected,
          empty: project.studies_count === 0,
        }
      )}
```

Reply:
or to pass one single object, but slightly meh

```
      className={cx(
       {
         'scenario-explorator-modal-part-itemslist-minicard': true,
         'project': true,
         'with-image': true,
          selected: isSelected,
          empty: project.studies_count === 0,
        }
      )}
```

Reply:
Hum I would rather keep it like this, I find it a bit long otherwise 
```
cx(
        'scenario-explorator-modal-part-itemslist-minicard study',
        {
          selected: isSelected,
          empty: study.scenarios_count === 0,
         }
      )}
```

Reply:
`scenario-explorator-modal-part-itemslist-minicard project with-image`

It’s really difficult to see that we have 3 different classes 
3 lines make it more obvious.
we’re not trying to have less lines, but to be more readable.

Reply:
I think the problem is to have more specific classnames and not too long.
In a future css oriented refacto we could rename those. The last Clara's proposition looks good to me ;)

Reply:
Agree on the fact that the 3 classes are a bit difficult to read and that the 1st class is especially not readable. I renamed it, do you think it is better now even if the 3 classes are on the same line ?